src/Fl_Image.cxx

```diff
@@ -383,8 +383,8 @@ Fl_RGB_Image::Fl_RGB_Image(const uchar *bits, int W, int H, int D, int LD) :
 /**
  The constructor creates a new image from the specified data.
 
- If the provided array is to small to contain all the image data, the
- constructor will not generate the image to avoid illegale memeory read
+ If the provided array is too small to contain all the image data, the
+ constructor will not generate the image to avoid illegal memeory read
```

Comment:
Sorry, "memeory" is still misspelled (at least three times in different files).

Reply:
Sorry, bad muscle memeory... ;-) . Fixed.